### ISSUE TITLE
How to validate indentation of prop values?

### ISSUE THREAD

**Issue by n1313** [yannickcr/eslint-plugin-react#1599]
OSX 10.12.6, eslint 4.9.0, eslint-plugin-react 7.4.0, babel-eslint 8.0.1.

This code does not trigger any warnings (with `jsx-indent` and `jsx-indent-props` set to ["error", 2]):

```
return (
  <Component
    prop={true}
    anotherProp={[
      {
        id: 1
      },
  {
        id: 2
  }
    ]}
  />
);
```

Is there a way to trigger a warning for mis-indented curly braces around second object in `anotherProp`? My expectation is that the value of `anotherProp` would be treated as a regular array of objects, triggering an `indent` warning from eslint (or `jsx-indent-props` from eslint-plugin-react).

**Comment by ljharb** [comment]
What's the setting for the `indent` rule?

**Comment by n1313** [comment]
@ljharb same, ["error", 2].

**Comment by ljharb** [comment]
hm, I do wonder if `JSXElement *` needs to be refined a bit. Just for testing, let's remove that one, and see what it complains about? My expectation is that it will fix the array in question but probably will break other things.

**Comment by n1313** [comment]
Indeed, setting `"indent": [ "error", 2, { "ignoredNodes": [ "JSXElement" ] }]` helped: invalid indentation in my test case is correctly identified and fixed.

**Comment by ljharb** [comment]
For now, that's a great workaround; would you mind filing this on airbnb/javascript? I don't think it's an issue with this repo.

**Comment by n1313** [comment]
Thanks, will do!
